src/node_buffer.cc

```diff
@@ -701,6 +931,53 @@ void IndexOfNumber(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void IsNull(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!HasInstance(args[0]))
+    return env->ThrowTypeError("first arg should be a Buffer");
+
+  CHECK(args[0]->IsObject());
+  ARGS_THIS(args[0].As<Object>())
+
+  uint32_t offset = args[1]->Uint32Value();
+  CHECK_LE(offset, obj_length);
+
+  char* ptr = obj_data + offset;
+
+  args.GetReturnValue().Set(ptr == nullptr);
```

Comment:
> That wasn't what I meant but I like that even better.

It's an idea I've been playing around with in my head for a while now… Why have `buffer.NULL` _and_ JS `null` when we could just map everywhere that C NULL could be returned to with JS `null`? I'm liking the idea the more that I think about it. Would also simplify things a bit API-wise.